packages/app/package.json

```diff
@@ -19,6 +19,10 @@
     "module": "dist/index.js",
     "types": "dist/index.d.ts",
     "exports": {
+      "./dist/style-lib.css": "./dist/style-lib.css",
+      "./style-lib.css": "./dist/style-lib.css",
+      "./dist/style.css": "./dist/style.css",
+      "./style.css": "./dist/style.css",
```

Comment:
I've added both import options for backward compatibility of the documentation. I don't want people who use webpack 5 and try to import with `/dist/` to get an error.

Reply:
Is there a point to shorten the paths then ?

Reply:
Oh, it's just prettier.